### ISSUE TITLE
Different order in sbt and gradle

### ISSUE THREAD

**Issue by jahan01** [liancheng/scalafix-organize-imports#135]
I am trying run the organize imports from various tools and I get different results.

```scala
import com.xxxx._
import scala.language.postfixOps
import scalikejdbc._
import scala.Ordering.Implicits._
```

**sbt-scalafix and metal's organize import:**
```scala
import scala.Ordering.Implicits._
import scalikejdbc._

import com.xxxx._

import scala.language.postfixOps
```

**gradle-scalafix output:**
```scala
import scala.Ordering.Implicits._
import scala.language.postfixOps
import scalikejdbc._

import com.xxxx._
```

And my `.scalafix.conf` file:
```
OrganizeImports {
  	groupedImports = Merge
	groupExplicitlyImportedImplicitsSeparately = true
	importsOrder = SymbolsFirst
}
``` 

First I thought this could be due to `gradle-scalafix` plugin issue but it appears that other scalafix rules correctly applied. I am puzzled and don't know in which repository to file the issue.

Looks like `groupExplicitlyImportedImplicitsSeparately` is not respected.

**Comment by liancheng** [comment]
@jahan01, thanks for reporting! What is the value of your `OrganizeImports.groups` option? The default one is

```
[
  "re:javax?\\."
  "scala."
  "*"
]
```

so neither output looks right to me (the `scalikejdbc.` import should not be grouped together with the `scala.` imports).

My hunch is that the Scalameta/SemanticsDB versions might differ in your sbt and Gradle builds. Is it possible to provide a minimal repro of this issue?

**Comment by jahan01** [comment]
`scalike` ordering is fine, I had a custom configuration for groups. Only thing missing is the new line between groups.
Anyways I have minimal project here which reproduces this bug and on the way have identified more bugs, details in the README.
https://github.com/jahan01/opensource-issues/tree/master/scalafix_organize_imports

Let me know if you need more details. Btw, I'm of MacOS 10.15 if that matters.

**Comment by liancheng** [comment]
@jahan01, thanks for the repro! I'm not familiar with Gradle but will try to find some time to investigate.

**Comment by liancheng** [comment]
Should have been fixed by cosmicsilence/gradle-scalafix#37.
